### PR TITLE
THIS IS MY SOS

### PR DESCRIPTION
**Pull request by javierbustillo**
WARNING I HAVE NO IDEA WHAT IM DOING

### CODE REVIEW

**Comment by javierbustillo** [comment]
https://www.npmjs.com/package/react-google-charts

**Comment by javierbustillo** [comment]
Is there a life after death

**Comment by javierbustillo** [comment]
One time I saw a dog get hit by a car. The old lady stopped the car got out and cried

**Comment by javierbustillo** [comment]
The remaining energy inside of my body is completely dedicated to not falling apart

**Comment by javierbustillo** [comment]
This is a cry for help. Everything is so dark

**Comment by ChristianPerez34** [comment]
> 
> 
> Is there a life after death

First we must define what life is. Life is the aspect of existence that processes, acts, reacts, evaluates, and evolves through growth (reproduction and metabolism). After knowing what life is, we must then define death. The concept of death has a use for the living, while death itself has no use for anything. All we can say about death is that it is either real or it is not real. If it is real, then the end of one's life is a simple termination.

**Comment by ChristianPerez34** [comment]
> 
> 
> One time I saw a dog get hit by a car. The old lady stopped the car got out and cried

This is a by product of human empathy

**Comment by ChristianPerez34** [comment]
> 
> 
> The remaining energy inside of my body is completely dedicated to not falling apart
You cannot fall further than the ground. Your energies will simply not permit this

**Comment by ChristianPerez34** [comment]
> 
> 
> This is a cry for help. Everything is so dark

Open your eyes

File: src/App.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ const App = () => {
             {/*TODO: Revert back for phase 3*/}
             <Route exact path="/" component={UserAuth}/>
             <Route path="/chats" component={ChatNavBar}/>
+            <Route path="/stats" component={Stats}/>
```
Comment (ChristianPerez34) [inline src/App.js:28]: Where is the import?

File: src/js/components/Stats.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        axios.get(`${process.env.REACT_APP_API_URL}` + 'stats/trending').then(
+            response => {
+                this.setState({trending_hashtags: response.data})
+            }
+        )
+    };
+
+    render() {
+        return (
+            <h1>Stats</h1>
```
Comment (ChristianPerez34) [inline src/js/components/Stats.jsx]: Should display Chart here using methods from ```react-google-charts```

**Review by ChristianPerez34 — CHANGES_REQUESTED** [review]

File: src/js/components/Stats.jsx (hunk 2 above)
Comment (javierbustillo) [reply src/js/components/Stats.jsx]: I  know my dude. I don't understand :(

**Comment by javierbustillo** [comment]
I once saw a pigeon slowly die after being stoned by children

File: src/js/components/Stats.jsx (hunk 2 above)
Comment (ChristianPerez34) [reply src/js/components/Stats.jsx]: Try using a table chart to make something similar to this. Without the get report thing 

Screenshot_2019-04-24 The Complete Guide to Hashtag Analytics

**Comment by ChristianPerez34** [comment]
> 
> 
> I once saw a pigeon slowly die after being stoned by children

Which one suffered most, the creature that was stoned to death or the powerless bystander that was not able to save said pigeon?

**Review by ChristianPerez34 — APPROVED** [review]

**Pull request merged**